tests/test_spearman_distance.py

```diff
@@ -30,8 +32,14 @@ def test_pairwise_distance_matrix():
     r2 = np.array([0.23, 0.24, 0.26, 0.27])
     r3 = np.array([0.22, 0.24, 0.26, 0.28])
 
-    rankings = [r1, r2, r3]
+    rankings = np.array([r1, r2, r3])
     D = pairwise_spearman_distance_matrix(rankings)
+    # Testing dask
+    client = Client()
+    D_dask = pairwise_spearman_distance_matrix(da.from_array(rankings))
+    client.close()
```

Comment:
Made a comment in `test_kendall_tau_distance.py ` about having a specific function for testing dask and would reccomend the same here. Remove the dask parts here and write a new `test_dask_pairwise_distance_matrix` function.